io.github.imbev.WhenPenguinsFly.metainfo.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Isaac Beverly -->
+<component type="desktop-application">
+  <id>io.github.imbev.WhenPenguinsFly</id>
```

Comment:
why `io.github` since it's hosted on codeberg ?

Reply:
Unless I misinterpreted the Flathub publishing documentation, repositories on github do not require a github pages site, however repositories on codeberg do.

There is a github mirror of the project.

Reply:
where? really this should match. It's not random.

Reply:
https://docs.flathub.org/docs/for-app-authors/verification#source-code-hosting-site

Codeberg doesn't appear to be considered a "Source code hosting site" yet

Reply:
No it works fine, there are several verified apps using `page.codeberg` ids. It probably needs to be updated.